src/lib/annotations/Annotator.js

```diff
@@ -212,12 +221,12 @@ class Annotator extends EventEmitter {
         }
 
         // Hide create annotations button if image is rotated
-        const annotateButton = this.previewUI.getAnnotateButton();
+        const pointAnnotateButton = this.previewUI.getAnnotateButton(SELECTOR_BOX_PREVIEW_BTN_ANNOTATE_POINT);
```

Comment:
pointAnnotateButton is then or null and as a result the annotatorUtil show and hide functions do nothing.